block.go

```diff
@@ -75,7 +75,9 @@ type IndexReader interface {
 	// The Postings here contain the offsets to the series inside the index.
 	// Found IDs are not strictly required to point to a valid Series, e.g. during
 	// background garbage collections.
-	Postings(name, value string) (index.Postings, error)
+	// 'reusePos' is the Postings object that can be re-used instead of allocating
+	// a new object. Re-use of this object need not be guaranteed.
+	Postings(name, value string, reusePos index.Postings) (index.Postings, error)
```

Comment:
`reusePostings` or `reusePosts` to be consistent and avoiding `Pos` which can be confused with `position`?

Reply:
Could you elaborate on ` Re-use of this object need not be guaranteed.` How this helps / warns caller? I am missing the meaning of this.

Reply:
> reusePostings or reusePosts to be consistent and avoiding Pos which can be confused with position?

Ah yes, I will change it. Thanks.

> Could you elaborate on Re-use of this object need not be guaranteed. How this helps / warns caller? I am missing the meaning of this.

I just meant to say that the implementation need not re-use this object, and it is fine to allocate a new object and return it instead. Maybe that sentence is not required at all as I say "_can_ be re-used" already.